util/util.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo"
+)
+
+func nowStamp() string {
+	return time.Now().Format(time.StampMilli)
+}
+
+func log(level string, format string, args ...interface{}) {
+	fmt.Fprintf(ginkgo.GinkgoWriter, nowStamp()+": "+level+": "+format+"\n", args...)
+}
+
+// Logf logs info log to ginkgo writer
+func Logf(format string, args ...interface{}) {
```

Comment:
Instead of Logf, we should have individual methods like Infof, Warnf, Errorf

Reply:
Updated in new incremental

Reply:
I was thinking of renaming Logf to Infof like the other methods or other logging libraries

Reply:
Yeah `Infof` makes more sense. Updated.